build.gradle

```diff
@@ -245,7 +245,8 @@ dependencies {
     }
     compile 'org.jooq:jool-java-8:0.9.14'
 
-    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.0.0'
+    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.0.0-ccd-1223.1'
+    testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '7.0.0-ccd-1131.7'
```

Comment:
either just a new prerelease that brings a new befta version, or a new befta release to be included as below. not both.

Reply:
for now I kept **testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '7.0.0-ccd-1131.7'**